src/main/java/org/fundacionjala/coding/jimmy/CesarCrypto.java

```diff
@@ -0,0 +1,149 @@
+package org.fundacionjala.coding.jimmy;
+
+
+/**
+ * Class for developing the methods.
+ */
+public class CesarCrypto {
```

Comment:
Split in two classes Cesar and Vigenere and apply abstraction for encode and decode